tasks/18.txt

```diff
@@ -1,3 +1,16 @@
 Create a SQL query to calcualte total orders count by each day in 1998:
 
-| Order Date | Total Number of Orders |
\ No newline at end of file
+| Order Date | Total Number of Orders |
+
+db.orders.aggregate([
+{$match: {OrderDate: {$gte: new Date(1998, 1, 1), $lt: new Date(1999, 1, 1)}}},
+{$group:{_id:"$OrderDate","TOC":{$sum:1}}},
+{$sort: {_id: 1}},
+{
+    $project:{
+        _id:0,
+        "OrderDate":{ $dateToString: { format: "%G-%m-%d", date: "$_id" }},
+        "Total Number of Orders":"$TOC"
+        }
+    },
+])
```

Comment:
It is ok for now, but please note date can be with time, so group by date in some situations can be wrong